tooling/api/src/tauri.ts

```diff
@@ -156,15 +173,16 @@ async function invoke<T>(cmd: string, args: InvokeArgs = {}): Promise<T> {
       cmd,
       callback,
       error,
-      ...args
+      payload: args,
+      options
     })
   })
 }
 
 /**
  * Convert a device file path to an URL that can be loaded by the webview.
  * Note that `asset:` and `https://asset.localhost` must be added to [`tauri.security.csp`](https://tauri.app/v1/api/config/#securityconfig.csp) in `tauri.conf.json`.
- * Example CSP value: `"csp": "default-src 'self'; img-src 'self' asset: https://asset.localhost"` to use the asset protocol on image sources.
+ * Example CSP value: `"csp": "default-src 'self' ipc:; img-src 'self' asset: https://asset.localhost"` to use the asset protocol on image sources.
```

Comment:
This should probably match the migration tool, which is also how assets is done.
```suggestion
 * Example CSP value: `"csp": "default-src 'self' ipc: https://ipc.localhost; img-src 'self' asset: https://asset.localhost"` to use the asset protocol on image sources.
```